### ISSUE TITLE
post-processor/vsphere: ovftool only shows error messages from STDERR

### ISSUE THREAD

**Issue by rickard-von-essen** [hashicorp/packer#6923]
For background see [Troubleshooting ovftool, unable to get specific errors](https://groups.google.com/d/msgid/packer-tool/536e527e-ba53-4e59-a8fc-9300661c5004%40googlegroups.com?utm_medium=email&utm_source=footer)

```golang
         if err := cmd.Run(); err != nil {
		err := fmt.Errorf("Error uploading virtual machine: %s\n%s\n", err, p.filterLog(errOut.String()))
		return nil, false, err
	}
```

If ovftool fails Packer only shows the STDERR, but it seems like at least some error messages are sent to STDOUT.

**Comment by rickard-von-essen** [comment]
There have been some related issues. I'll try to link them later when today.

**Comment by rickard-von-essen** [comment]
Related to #5499 and #3321

**Comment by spawnia** [comment]
I have an example. Here is what Packer gives me:

```
Build 'vmware-iso' errored: 1 error(s) occurred:
* Post-processor failed: Error uploading virtual machine: exit status 1
```

Whereas calling `ovftool` manually yields the following output:

```
Opening OVF source: ovf_back/packer-centos/packer-centos.ovf
The manifest validates
Opening VI target: vi://packer%40vs-mll.local@192.168.0.220:443/MLL/host/MLL-Cluster
Error: Fault cause: vim.fault.NoPermission

Completed with errors
```

Bubbling up that message would be really helpful.

**Comment by nywilken** [comment]
Hi @spawnia, thanks for diving into the error messaging. I'm not sure if we will be able to get to this soon. But would be happy to review and help push through a pull request if you're interested in making the change.

**Comment by SwampDragons** [comment]
This error messaging is fixed in #9649; the stdout will now be piped directly to the UI.
